supporting_scripts/analysis-of-endpoint-connections/src/main/typeScript/Postprocessor.ts

```diff
@@ -108,10 +108,10 @@ export class Postprocessor {
                                         urlEvaluationResult = this.evaluateUrl(node.arguments[0], methodDefinition, node, classBody);
                                     }
 
-                                    const fileName = this.fileName;
+                                    const filePath = this.filePath;
                                     if (urlEvaluationResult.resultType === ParsingResultType.EVALUATE_URL_SUCCESS) {
                                         for (let url of urlEvaluationResult.result) {
-                                            this.restCalls.push({ method, url, line, fileName });
+                                            this.restCalls.push({ method, url, line, filePath: filePath });
```

Comment:
If the name of the key is already `filePath`, I think you can just write:

```suggestion
                                            this.restCalls.push({ method, url, line, filePath });
```